conjur_api/http/ssl/ssl_context_factory.py

```diff
@@ -55,12 +55,18 @@ def _get_mac_ca_certs() -> str:
     logging.debug("Get CA certs from mac keychain")
 
     try:
-        get_ca_certs_process = subprocess.run(
+        get_root_ca_certs_process = subprocess.run(
             ["security", "find-certificate", "-a", "-p", "/System/Library/Keychains/SystemRootCertificates.keychain"],
             capture_output=True,
             timeout=10,
             check=True,
             text=True)
-        return get_ca_certs_process.stdout
+        get_system_ca_certs_process = subprocess.run(
+            ["security", "find-certificate", "-a", "-p", "/Library/Keychains/System.keychain"],
+            capture_output=True,
+            timeout=10,
+            check=True,
+            text=True)
+        return get_root_ca_certs_process.stdout + get_system_ca_certs_process.stdout
```

Comment:
Will this import certs that are in the System keychain but are not set as trusted? See this [comment](https://gist.github.com/gl-johnson/a6a926953c436b54c2741a23cd87e144?permalink_comment_id=4287179#gistcomment-4287179) from @andytinkham on @gl-johnson's gist.

Reply:
It was added because in Cyberark network (and many other companies internal networks I guess) the certificate for outgoing requests is replaced with internal one which is stored in system keychain.
BTW we pull certificates from trusted root keychain without validating their trust too.

Reply:
Hmm. Any way we can validate trust before pulling in these certs?
CC @andytinkham, @gl-johnson

Reply:
It may be safe to assume that the `System Root` keychain contains only trusted root CA's, but the `System` keychain can be easily modified and therefore is more likely to contain self-signed certs or regular certificates which are not explicitly trusted on the system. In this case the current command would still pull them into the SSL context object we are using to validate the connection.

There is a `security verify-cert` subcommand which may be useful for validating the certificates actually have the correct trust settings before we append them to the SSL context. [Example here.](https://gist.github.com/mjdetullio/592239b0fe3988a7e6ad)

Reply:
@rafis3 Any update on this issue?

Reply:
OK great!